libs/input-mapping/tests/Helper/BuildQueryFromConfigurationHelperTest.php

```diff
@@ -117,6 +117,19 @@ public function testBuildQuerySourceTagsAndQuery(): void
         );
     }
 
+    public function testBuildQuerySourceTagsAndQueryWhenSourceTagsEmpty(): void
+    {
+        self::assertEquals(
```

Comment:
Ještě možná přidat test s prázdným `source` klíčem?

Reply:
To je ošetřené tím novým checkem a tudíž i tím testem, pokud bude source prázdný nebo nenasetovaný, tak se to nikdy nedostane na tags klíč.